Makefile

```diff
@@ -0,0 +1,10 @@
+build:
```

Comment:
Can you open this as an issue? As above, I'll assign it to a release if we decide to go that way.